ui/src/layouts/MainLayout.vue

```diff
@@ -53,63 +80,61 @@
 
 <script lang="ts">
 
-import { defineComponent, ref } from 'vue'
-import { useI18n } from 'vue-i18n'
 import MenuItems from 'components/MenuItems.vue'
-import Restart from 'src/components/Reboot.vue'
+import menuList from '../components/styles/menuList'
+import { qHeaderStyle } from '../components/styles/qStyles'
+import { useQuasar } from 'quasar'
+import Reboot from 'src/components/Reboot.vue'
 import Shutdown from 'src/components/Shutdown.vue'
+import { defineComponent, ref, watch } from 'vue'
+import { useI18n } from 'vue-i18n'
+import type { QuasarLanguage } from 'quasar'
 
 export default defineComponent({
   name: 'MainLayout',
 
   components: {
     MenuItems,
-    Restart,
+    Reboot,
     Shutdown
   },
 
   setup () {
+    const $q = useQuasar()
+
     const leftDrawerOpen = ref(false)
-    // eslint-disable-next-line @typescript-eslint/unbound-method
-    const { t } = useI18n()
-
-    const menuList = [
-      {
-        icon: 'visibility',
-        label: t('Device_Info'),
-        path: 'home'
-      },
-      {
-        icon: 'book',
-        label: t('Documentation'),
-        path: 'documentation'
-      },
-      {
-        icon: 'router',
-        label: t('Networking'),
-        path: 'networking'
-      },
-      {
-        icon: 'update',
-        label: t('Update'),
-        path: 'update'
-      },
-      {
-        icon: 'settings',
-        label: t('Settings'),
-        path: 'settings'
-      }
-    ]
+    const { locale } = useI18n({ useScope: 'global' })
 
-    // A simple redirect to another page
-    function redirect (path: string) {
-      window.open(path)
+    // Quasar requires the Quasar language pack to be set seperate from Vue i18n
+    watch(locale, val => {
+      // Dynamic import, so loading on demand only
+      import(
+        // Set which Quasar language packs to import below:
+        // https://quasar.dev/options/quasar-language-packs#dynamical-non-ssr-
+        /* webpackInclude: /(de|en-US)\.js$/ */
+        'quasar/lang/' + val
+      ).then(lang => {
+        $q.lang.set(lang.default as QuasarLanguage)
+        $q.localStorage.set('lang', lang.default.isoName)
```

Comment:
This currently has a bug in Quasar which means refreshing the page still won't remember what the previous set language is. When the bug is fixed in Quasar I will update the Quasar package and then this will work.

Reply:
Resolved, it was a typo I had made.